packages/oasgraph-cli/README.md

```diff
@@ -19,29 +19,49 @@ npm i -g oasgraph-cli
 
 
 ## Usage
-The basic usage of the CLI takes the specified OAS, create a GraphQL interface for it, and start a server to host the GraphQL interface.
 
-You can specify a local file containing the OAS specification or a remote url such as `http://127.0.0.1:3000/openapi.json`. Additionally, you can specify an optional port number so you can have multiple GraphQL servers for testing in the same machine.
+```
+Usage: oasgraph <OAS JSON file path or remote url> [options]
+
+Options:
+  -V, --version             output the version number
+  -p, --port <port>         select the port where the server will start
+  -s, --strict              create a GraphQL schema matching the given OAS perfectly
```

Comment:
I am not sure about the wording: wouldn't the user expect a "perfectly" matched schema? Maybe we need a little more explanation here.

Reply:
Would something along the lines of `throw an error if OASGraph cannot create a GraphQL schema matching the given OAS perfectly` be better? 

Here is the [options doc](https://github.com/strongloop/oasgraph/blob/master/packages/oasgraph/README.md#options)

Reply:
I like it better